.oscal/ssp.oscal.yaml

```diff
@@ -0,0 +1,162 @@
+system-security-plan:
+  uuid: 8c1ee2a6-bd14-4f77-b2f7-3d2bc0aaa55e
+  metadata:
+    title: "Oscal Server Software Development System Security Plan"
+    last-modified: "2024-10-24T00:00:00Z"
+    version: "1.0"
+    oscal-version: "1.0.4"
+    parties:
+      - uuid: 11111111-0000-4000-9000-100000000001
+        type: organization
+        name: "Metaschema Framework Orginization"
+        short-name: "MFO"
+    roles:
+      - id: system-owner
+        title: System Owner
+      - id: security-officer 
+        title: Security Officer
+    responsible-parties:
+      - role-id: system-owner
+        party-uuids:
+          - 11111111-0000-4000-9000-100000000001
+      - role-id: security-officer
+        party-uuids:
+          - 11111111-0000-4000-9000-100000000001
+
+  import-profile:
+    href: "#"
+    include-controls:
+      - with-ids:
+          - po1
+          - po2
+          - po3
+          - po4
+          - po5
+          - ps1
+          - ps2
+          - ps3
+          - pw1
+          - pw2
+          - pw4
+          - pw5
+          - pw6
+          - pw7
+          - pw8
+          - pw9
+          - rv1
+          - rv2
+          - rv3
```

Comment:
Blocking: Can you do this in a SSP?

https://pages.nist.gov/OSCAL-Reference/models/v1.1.2/system-security-plan/json-outline/